lib/services_cli.rb

```diff
@@ -25,12 +25,12 @@ def root?
 
     # Current user running `[sudo] brew services`.
     def user
-      @user ||= Utils.safe_popen_read("/usr/bin/whoami").chomp
+      @user ||= ENV["USER"]
```

Comment:
Weirdly this isn't always set so may want to fall-back here with
```suggestion
      @user ||= ENV["USER"].presence || Utils.safe_popen_read("/usr/bin/whoami").chomp
```